velox/exec/Window.cpp

```diff
@@ -53,35 +30,10 @@ Window::Window(
           windowNode->id(),
           "Window"),
       numInputColumns_(windowNode->sources()[0]->outputType()->size()),
-      data_(std::make_unique<RowContainer>(
-          windowNode->sources()[0]->outputType()->children(),
-          pool())),
-      decodedInputVectors_(numInputColumns_),
+      windowBuild_(std::make_unique<SortWindowBuild>(windowNode, pool())),
```

Comment:
Does this PR enable StreamingWindow? It seems only Sort Window is enabled.

Reply:
@JkSelf : This PR doesn't enable StreamingWindow. The intent was for your team to enable StreamingWindowBuild. Though I have put in a sample code for you https://github.com/facebookincubator/velox/pull/6212

Reply:
Sure. I will enable StreamingWindow after this PR merged. Thanks.

Reply:
Thanks @JkSelf